contexts/utopia-natcert/v2.json

```
@@ -0,0 +1,21 @@
+{
+  "@context": {
+    "@protected": true,
+    "confidenceMethod": {
+      "@id": "https://www.w3.org/2018/credentials#confidenceMethod",
+      "@type": "@id"
+    },
+
+    "ConfidenceMethod": {
+      "@protected": true,
+      "@id": "https://www.w3.org/2018/credentials#ConfidenceMethod",
+      "@context": {
+        "controller": {
+          "@id": "https://w3id.org/security#controller",
+          "@type": "@id"
+        }
+      }
+    },
+    "DidAuthnConfidenceMethod": "https://examples.vcplayground.org/contexts/utopia-natcert/vocab#DidAuthnConfidenceMethod"
```

Would a future real vocab URL be similar to the birth cert one?
```suggestion
    "DidAuthnConfidenceMethod": "https://examples.vcplayground.org/vocabs/utopia-natcert#DidAuthnConfidenceMethod"
```